api/templates/location-city-state-zipcode-country.xml

```diff
@@ -0,0 +1,4 @@
+<City>{{.props.city}}</City>
+<State>{{toUpper .props.state}}</State>
+<ZipCode>{{.props.zipcode}}</ZipCode>
+<Country>{{.props.country}}</Country>
```

Comment:
This needs to pull `.props.county` into `<County>`

Reply:
Forgot to add that would be best to rename the file so as not to reference `country` but `county`.

Reply:
Should we remove `<Country>` from here then? It is assumed to be the US unless it's specifically a foreign country, right?

Reply:
Yeah, sorry – no `Country` element in this one.